docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -5,3 +5,57 @@
 * `selenium.grid.username` - defines remote grid username
 * `selenium.grid.password` - defines remote grid password
 * `selenium.grid.host` - defines remote grid hostname
+
+== Known Issues
+
+Known issues mechanism allows separate fails and known issues existing within the project.
```

Comment:
`allows to distinguish failures from known issues presented in the system under test.`